app/konnect/updates.md

```diff
@@ -9,6 +9,21 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
+## February 2024
+
+**Additional plugin support for consumer groups**
+: Along with {{site.base_gateway}} 3.6 support in {{site.konnect_short_name}}, additional plugins are now supported for consumer groups.
+
+: The following plugins can be applied to consumer groups using the Admin API and the Gateway Manager UI:
+* [Rate Limiting (OSS)](/hub/kong-inc/rate-limiting/)
+* [Request Termination](/hub/kong-inc/request-termination/)
+* [Proxy Cache](/hub/kong-inc/proxy-cache/)
+* [Proxy Cache Advanced](/hub/kong-inc/proxy-cache-advanced/)
+* Improved [ACL plugin](/hub/kong-inc/acl/)
```

Comment:
@hl782 Hi! I pulled this list of supported plugins from the brief, and I just wanted to verify with you that this list is correct. Thanks!